src/meta/src/hummock/hummock_manager.rs

```diff
@@ -738,21 +733,18 @@ where
 
             // Commit tables by moving them into level0
             let version_first_level = new_hummock_version.levels.first_mut().unwrap();
-            match version_first_level.get_level_type()? {
-                LevelType::Overlapping => {
-                    uncommitted_epoch
-                        .tables
-                        .iter()
-                        .for_each(|t| version_first_level.table_infos.push(t.clone()));
-                }
-                LevelType::Nonoverlapping => {
-                    return Err(ErrorCode::NotImplemented(
-                        "unsupported LevelType::Nonoverlapping".to_string(),
-                        None.into(),
-                    )
-                    .into())
-                }
-            };
+            if version_first_level.level_idx == 0 {
```

Comment:
Seems we can still use LevelType and these code can remain unchanged?

Reply:
The Level 0 is special and it must be overlap.
The Level 1 may be overlap because it may be distributed by hash and the key range between each can be overlap.